tikv-client/src/main/java/com/pingcap/tikv/codec/CodecDataInput.java

```diff
@@ -327,7 +327,7 @@ public long skip(long n) {
         k = n < 0 ? 0 : n;
       }
 
-      pos += k;
+      pos = Math.toIntExact(pos + k);
```

Comment:
pos + k will not overflow for it always < count which is int type

Reply:
it can only be changed to type strong transfer

Reply:
you can set k to int type, and `k = n < 0 ? 0 : (int)n`